content/code-style.md

```diff
@@ -99,25 +99,11 @@ You can also add a `eslintConfig` section to your `package.json` to specify that
 {
   ...
   "scripts": {
-    "lint": "eslint .",
+    "lint": "./node_modules/.bin/eslint .",
```

Comment:
This shouldn't have to call `./node_modules/.bin` as that should be set automatically by the `npm` command when running `meteor npm run lint`.